### PR TITLE
allow to make fulltextsearch:index issues easier to debug

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/all-in-one/discussions/1694
Close https://github.com/nextcloud/all-in-one/pull/1692

- [x] TODO: fix the todo

Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

File: Containers/nextcloud/entrypoint.sh [hunk 1]
```diff
@@ -592,10 +592,11 @@ if [ "$FULLTEXTSEARCH_ENABLED" = 'yes' ]; then
         echo "Waiting 10s before activating FTS..."
         sleep 10
         echo "Activating fulltextsearch..."
-        if php /var/www/html/occ fulltextsearch:test && php /var/www/html/occ fulltextsearch:index; then
+        if php /var/www/html/occ fulltextsearch:test && php /var/www/html/occ fulltextsearch:index --no-readline; then
             touch "$NEXTCLOUD_DATA_DIR/fts-index.done"
         else
             echo "Fulltextsearch failed. Could not index."
+            # TODO add link to wiki article what to do when the index fails regularly
```
Comment (szaimen) [inline Containers/nextcloud/entrypoint.sh:599]: todo

Comment (szaimen) [reply Containers/nextcloud/entrypoint.sh]: done

**Comment by szaimen** [comment]
@ManOki does this look good to you?

**Review by ManOki — APPROVED** [review]
LGTM

**Comment by szaimen** [comment]
Thanks for the approval! :)

**Pull request merged**